### ISSUE TITLE
Inconsistent annotation documentation for k8s service objects

### ISSUE THREAD

**Issue by Deltahedron** [kubernetes-sigs/aws-load-balancer-controller#3776]
**Is your feature request related to a problem?**
Going through the documentation it can be confusing to find the correct annotations for k8s objects. The documentation [page](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/) is usually quite clear and good for initial setup but once there's a need to modify the defaults it's not clear which source should be trusted and leads to additional Google searches and trial and error approach which could be easily avoided.

In this instance I was trying to limit the subnets used for a particular LB and I knew it was supported but finding the correct annotation or place in the docs was not clear.

It may have been my fault but the annotation list split in the docs (Ingress/Service), having to click through different sections and pages and the page search not clearly returning a simple list is a drawback.

**Describe the solution you'd like**
Perhaps the annotation list could be joined or displayed in a way where it's easy to search through all of the available annotations would be more beneficial for discovery and deciding which one to go for.

Additionally the subnet discovery page doesn't mention the fact that subnets can be manually configured with a link to the annotations which would be great:
[subnet discovery](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/deploy/subnet_discovery/)
[subnet annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/guide/service/annotations/#subnets)

The search in the docs page could be improved but that's beyond the scope I'd guess.

Anyway, thanks for a great project and support!

**Comment by M00nF1sh** [comment]
/kind documentation
Thanks for the valuable feedback.
 just for clarification, do you mean organize the doc thus configurations for similar LB settings(e.g. subnets) for Ingress/Service are listed together?

**Comment by Deltahedron** [comment]
@M00nF1sh yes, I think this would be valuable as it makes feature discovery and looking for specific configs more streamlined.

**Comment by alliasgher** [comment]
Opened #4707 for this.
